qa/acceptance/spec/lib/cli/logstash-plugin/install_spec.rb

```diff
@@ -0,0 +1,74 @@
+# encoding: utf-8
+require_relative "../../../spec_helper"
+require "logstash/version"
+require "fileutils"
+require "logstash/devutils/rake"
```

Comment:
I know where the error is coming from, I'm not running this exemples with rake, so the whole rake context is not in. As a syntax like:  ```bundle exec rspec acceptance/spec/lib/cli/logstash-plugin/install_spec.rb``` should be possible I do recommend we use code here that is not rake dependant.